.build-harness

```diff
@@ -0,0 +1,19 @@
+export SHELL = /bin/bash
```

Comment:
This file and the build-harness/ directory should be in `.gitignore` and not committed into the repo.